core/tests/protractor_desktop/navigation.js

```diff
@@ -220,6 +220,13 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
 
+  it('should visit the Volunteer page', async function() {
+    await browser.get('/volunteer');
+    await waitFor.pageToFullyLoad();
+    expect(await element(
```

Comment:
I think we should wait for the element to be visible (`waitFor.visibilityOf()`) before/instead of expecting this. I feel this could very easily turn into a flake/failure

Reply:
Done.